src/common_utils/database_preparing/set_init_configure.py

```diff
@@ -52,9 +53,12 @@ def contains(key, parameter):
 def main():
     db = DatabaseProxy('mongodb://localhost:27017/', 'PepperChatDB')
     create_collections(db)
-    insert_polish_stop_words(db)
-    scrapper_data.initialize_main_collection_from_scrapper(db)
-    scrapper_data.inert_into_database('./csv_files/main_statements.csv', db)
+    initialize_polish_stopwords_collection(db)
+    csv_data.initialize_main_collection_from_scrapper(db)
+    csv_data.insert_into_database('./csv_files/main_statements.csv', db)
+    csv_data.insert_into_database('./csv_files/numbers_log_adapter_init_data.csv', db, conf.NUMBERS_QUEST_COLLECTION.value)
```

Comment:
Why this is not in main?

Reply:
because this is script for initializing database, invoked in initialize_db.sh, main.py is another piece of code (cake) :D